Content.Client/Electrocution/ElectrocutionHUDVisualizerSystem.cs

```diff
@@ -70,10 +68,9 @@ private void ShowHUD()
     private void RemoveHUD()
     {
         var electrifiedQuery = AllEntityQuery<ElectrocutionHUDVisualsComponent, AppearanceComponent, SpriteComponent>();
-        while (electrifiedQuery.MoveNext(out var uid, out var _, out var appearanceComp, out var spriteComp))
+        while (electrifiedQuery.MoveNext(out var uid, out var _, out _, out var spriteComp))
```

Comment:
```suggestion
        while (electrifiedQuery.MoveNext(out var uid, out _, out _, out var spriteComp))
```
same for the other one above